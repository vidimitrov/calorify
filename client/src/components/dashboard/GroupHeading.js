import styled from 'styled-components';

const InputWrapper = styled.h2`
  color: #75736d;
  margin-top: 0px;
  font-size: 24px;

  @media (max-width: 420px) {
    font-size: 16px;
  }
`;

export default InputWrapper;
