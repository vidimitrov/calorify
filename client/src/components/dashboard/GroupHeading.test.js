import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import GroupHeading from './GroupHeading';

it('Dashboard/GroupHeading component renders correctly', () => {
  const tree = renderer.create(<GroupHeading>Group 1</GroupHeading>).toJSON();
  expect(tree).toMatchSnapshot();
});
