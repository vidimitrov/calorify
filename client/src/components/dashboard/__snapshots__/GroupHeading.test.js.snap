// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/GroupHeading component renders correctly 1`] = `
.c0 {
  color: #75736d;
  margin-top: 0px;
}

<h2
  className="c0"
>
  Group 1
</h2>
`;
